### PR TITLE
Menue link component

### LINKED ISSUE #9: MenueLink component
# Design

Screenshot 2021-09-11 at 20 50 31

- Big Buttons with link to an other story

Color:
Fill rgba(230, 241, 243, 0.25)

Drop shadow:
rgba(0, 0, 0, 0.25)  blur 4

width 315px
height 71px

Typography size="l"

Figma:
https://www.figma.com/file/JSwurYCMP9Lt1OUEISErxw/Magda-Gesellenst%C3%BCck-Version-2?node-id=67%3A203

### PR DESCRIPTION
**Pull request by MagdaBer**
- Create menue link function and story

- Add styling
I am not sure if it is ok to nest a div tag inside the link tag. 

- Closes #9

### CODE REVIEW

File: src/app/components/MenueLink/MenueLink.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import Typography from '../Typography/Typography';
+import styles from './MenueLink.module.css';
+
+export type MenueLinkProps = {
+  children: ReactNode;
+  to: string;
+  className?: string;
+};
+
+function MenueLink({ children, to, className }: MenueLinkProps): JSX.Element {
```
Comment (marwinburesch) [inline src/app/components/MenueLink/MenueLink.tsx:13]: 🧐 I believe 'menu' is spelled incorrectly

File: src/app/components/MenueLink/MenueLink.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  component: MenueLink,
+  argTypes: {},
+};
+
+const Template: Story<MenueLinkProps> = (args) => <MenueLink {...args} />;
+
+export const Sprossenfinder = Template.bind({});
+Sprossenfinder.args = {
+  children: 'Sprossenfinder',
+  to: '/app/components/sprossenfinder',
```
Comment (marwinburesch) [inline src/app/components/MenueLink/MenueLink.stories.tsx:17]: You would use a link like this in your app, but that doesn't matter here, in storybook

**Review by marwinburesch — COMMENTED** [review]
Nice!

File: src/app/components/MenueLink/MenueLink.tsx (hunk 1 above)
Comment (MagdaBer) [reply src/app/components/MenueLink/MenueLink.tsx:13]: Oops...

**Pull request merged**